### ISSUE TITLE
support client certificates

### ISSUE THREAD

**Issue by boringcactus** [LukeEmmet/GemiNaut#9]
this is probably gonna suck to implement, but it'd be nice to have eventually for things like [gemifedi](https://git.sr.ht/~boringcactus/gemifedi)

**Comment by boringcactus** [comment]
probably blocked on gemget implementing the same feature

**Comment by LukeEmmet** [comment]
Yes, when Gemget supports them GemiNaut will get them. I think it is on the gemget roadmap

**Comment by makeworld-the-better-one** [comment]
Hi, I'm the gemget developer. Pretty busy at the moment, but I would like to add that feature yeah. Would happily accept any PRs as well! You would need to add cert support to the underlying [go-gemini](https://github.com/makeworld-the-better-one/go-gemini) first.

Shouldn't be too hard overall, so I will try and do this when I can. Happy to hear there's interest!

**Comment by makeworld-the-better-one** [comment]
gemget [v1.6.0](https://github.com/makeworld-the-better-one/gemget/releases/tag/v1.6.0) has been released! It supports client certs. I hope this helps GemiNaut. :smile:

**Comment by LukeEmmet** [comment]
Hi @makeworld-the-better-one - thanks for the update about the new gemget release.  That is great news that client certs are now supported. I will investigate further.

**Comment by LukeEmmet** [comment]
closing this ticket - client certificates are implemented since 0.8.11
